internal/llrp/reader_test.go

```diff
@@ -392,8 +392,10 @@ func TestClient_ManySenders(t *testing.T) {
 		go func() {
 			defer msgGrp.Done()
 
+			//nolint: gosec
 			sz := rand.Int31n(1024)
 			data := make([]byte, sz)
+			//nolint: gosec
```

Comment:
What does linter no like in these?

Reply:
```
internal/llrp/reader_test.go:395:10: G404: Use of weak random number generator (math/rand instead of crypto/rand) (gosec)
                        sz := rand.Int31n(1024)
```